shell/common/native_mate_converters/value_converter.h

```diff
@@ -44,7 +44,7 @@ struct Converter<base::Optional<T>> {
       return true;
     }
     T converted;
-    if (Converter<T>::FromV8(isolate, val, &converted)) {
+    if (!Converter<T>::FromV8(isolate, val, &converted)) {
```

Comment:
it's unlikely to affect usages, but we should also set the optional to empty when we get undefined/null, rather than leaving it as it was.
```suggestion
    if (!Converter<T>::FromV8(isolate, val, &converted)) {
      *out = base::nullopt;
```

Reply:
Did this in both places 👍